rfhub2/cli/rfhub_importer.py

```diff
@@ -0,0 +1,216 @@
+from pathlib import Path
+import re
+from robot.errors import DataError
+from robot.libdocpkg import LibraryDocumentation
+import robot.libraries
+from typing import Dict, List, Set, Tuple
+
+from .api_client import Client
+from rfhub2.model import Collection, Keyword
+
+
+RESOURCE_PATTERNS = {'.robot', '.txt', '.tsv', '.resource'}
+ALL_PATTERNS = (RESOURCE_PATTERNS | {'.xml', '.py'})
+EXCLUDED_LIBRARIES = {'remote.py', 'reserved.py', 'dialogs.py', 'dialogs_jy.py',
+                      'dialogs_py.py', 'dialogs_ipy.py', 'setup.py'}
+INIT_FILES = {'__init__.txt', '__init__.robot', '__init__.html', '__init__.tsv'}
+
+
+class RfhubImporter(object):
+
+    def __init__(self, paths: Tuple[Path, ...], no_installed_keywords: bool, client: Client) -> None:
+        self.paths = paths
+        self.no_installed_keywords = no_installed_keywords
+        self.client = client
+
+    def delete_collections(self) -> Set[int]:
+        """
+        Deletes all existing collections.
+        """
+        self.client.check_communication_with_app()
+        collections = self.client.get_collections()
```

Comment:
it is worth to remember about paging the results here, but I think we can skip it for now.

Reply:
Yes, I know about it, and agree to keep it for later :)